.github/config/versions.json

```diff
@@ -0,0 +1,5 @@
+{
+  "release/23.3.x": "23.3.9",
+  "release/24.1.x": "24.1.4",
+  "release/24.3.x": "24.1.4"
```

Comment:
I think it is not a typo on line 4: 24.1.4

Maybe you can bump all the versions to the last MR